addons/ai/XEH_PREP.hpp

```diff
@@ -1,9 +1,7 @@
-PREP(assignLoadout);
-PREP(selectItem);
-PREP(selectAttachments);
-PREP(garrison);
-PREP(handleDebugTags);
-PREP(sys_stance);
-PREP(handleLocality);
-PREP(pushLocalCommand);
-PREP(handleWaypointPlaced);
```

Comment:
This seems to be the shift+click to force AI traversal with careless combat stance. I do not recommend taking that out with the other changes.

Reply:
Yeah it is, and it is simply way too brute force IMO.

```
_unit setUnitPos "UP";
_unit setSpeedMode "FULL";
_unit setBehaviour "AWARE";
_unit setCombatMode "BLUE";

_unit disableAI "TARGET";
_unit disableAI "AUTOTARGET";

_unit allowFleeing 0;

_unit doWatch objNull;
_unit doTarget objNull;
_unit commandWatch objNull;
_unit commandTarget objNull;
```
Especially seeing that there is no notification or indication whenever you're using it.